CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ x.x.x Release notes (yyyy-MM-dd)
   cannot exist independently from a `Realm`.
 * Aggregate operations are now available on `List`: `min`, `max`, `sum`,
   `average`.
+* Committing write transactions (via `commitWrite` / `commitWriteTransaction` and 
+  `write` / `transactionWithBlock`) now optionally allow for handling errors when
```

Comment:
It's non-breaking & entirely optional for ObjC/Swift 1.2, but I can still move it if you like